numpy/polynomial/hermite_e.py

```diff
@@ -1604,6 +1605,10 @@ def hermecompanion(c):
     return mat
 
 
+def hermeinterp(func, deg, x=None, args=(), window=None):
+    return NotImplemented
```

Comment:
This is not what `NotImplemented` is for. You could use `raise NotImplementedError`, but if you're going to do that, you might as well just do it in the base class